src/de/willuhn/jameica/hbci/passports/pintan/ChipTANDialog.java

```diff
@@ -211,7 +211,7 @@ protected void paint(final Composite parent) throws Exception
     
     if (this.usb)
     {
-      Application.getMessagingFactory().sendMessage(new StatusBarMessage(i18n.tr("Legen Sie die bitte Chipkarte ein."),StatusBarMessage.TYPE_INFO));
+      Application.getMessagingFactory().sendMessage(new StatusBarMessage(i18n.tr("Legen Sie bitte die Chipkarte ein."),StatusBarMessage.TYPE_INFO));
```

Comment:
Anmerkung: Ich habe in den i18n-Dateien den originalen Text aus den ChipTanDialog nicht gefunden, es muss also vermutlich dort nichts angepasst werden.